File: 프로그래머스/lv1/12950. 행렬의 덧셈/행렬의 덧셈.js
function solution(arr1, arr2) {
    var answer = [];
    var y = arr1.length;
    var x = arr1[0].length;
    
    for (var i = 0; i < y; i++){
        var temp = [];
        for (var j = 0; j < x; j++){
            temp.push(arr1[i][j] + arr2[i][j]); 
        }
        answer.push(temp);
    }
    return answer;
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int y = arr1.length;
        int x = arr1[0].length;
        int[][] arr = new int[y][x];
        
        for (int i = 0; i < y; i++) {
            for (int j = 0; j < x; j++) {
               arr[i][j] = arr1[i][j] + arr2[i][j]; 
            }
        }
        return arr;
    }
}